src/sync/partial_sync.cpp

```diff
@@ -16,6 +16,9 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+#define __STDC_LIMIT_MACROS // See https://stackoverflow.com/a/3233069/1389357
+#include <cstdint>
```

Comment:
These includes should come after the realm ones (the order used in OS files is corresponding header -> local headers -> core/sync headers -> system headers).